_includes/_card.html

```diff
@@ -0,0 +1,9 @@
+<div class="card">  
+  <a href="{{ item.link_url }}" class="block">

```

Comment:
this should be `item.url` and I think we will want this anchor to span the whole click area.